cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,33 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io/#/login');
+  });
+
   it('should provide an ability to log in', () => {
+    const userName = `matesobaka@mate.ua`;
```

Comment:
Change the name of the variable userName to email, since the email address is used for login.